docs/conf.py

```diff
@@ -105,17 +106,16 @@
 html_title = "Stravalib Docs"
 # Theme and css
 html_theme = "pydata_sphinx_theme"
-# Add edit button to furo theme
 
 # Link to our repo for easy PR/ editing
 html_theme_options = {
-    "header_links_before_dropdown": 4,
+    "header_links_before_dropdown": 5,
     "use_edit_page_button": True,
     "show_toc_level": 1,
     # "navbar_align": "left",  # [left, content, right] For testing that the navbar items align properly
     "github_url": "https://github.com/stravalib/stravalib",
     "footer_start": ["copyright"],
-    "announcement": "<a href='stravalib-2.html'>Stravalib 2.x is out 🚀! Check out our migration guide for tips on changes from Stravalib V1!</a>",
+    "announcement": "<a href='/whats-new/stravalib-2.html'>Stravalib 2.x is out 🚀! Check out our migration guide for tips on changes from Stravalib V1!</a>",
```

Comment:
The banner link was broken because i moved the file and my build wasn't clean. it's working again now. i fixed a few other issues with the changelog as well.

Reply:
Awesome, I see it now. I can add the content of that document sometime this week.